cdrom/aulas/templates/aulas/aula_edit.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% block site_content %}
+
+<h1>Editar Aula</h1>
+<form method="post">
+	{% csrf_token %}
+	{{ form.as_p }}
+
+	<!-- Si hay características en el aula, puedes mostrarlas aquí -->
+	{% for caracteristica_form in formset %}
+		{{ caracteristica_form.as_table }}
+	{% endfor %}
+
+	<!-- Si hay fotos asociadas al aula, puedes mostrarlas aquí -->
+	{% for foto_form in formset_fotos %}
+		{{ foto_form.as_table }}
+	{% endfor %}
+
+	<input type="submit" value="Guardar cambios">
+</form>
+<a href="{% url 'aula_detail' aula.pk %}">Cancelar</a>
```

Comment:
```suggestion
	<a type="submit" role="button" class="btn btn-primary" href="{% url 'aula_detail' aula.pk %}">Guardar</a>
	<a role="button" class="btn btn-warning" href="{% url 'aula_detail' aula.pk %}">Cancelar</a>
</form>
```

![image](https://github.com/avdata99/clasdrom/assets/3237309/f1d21b6d-b888-4fd8-94ea-6804c180f5bc)